### ISSUE TITLE
USWDS-Site - Feature: Add "Show all" link under Showcase sites on homepage

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-site#2725]
### Is your feature request related to a problem? Please describe.

During [top tasks usability testing](https://docs.google.com/document/d/1snexeKkagu5isGupYIEICzPOVcYhXXnkfIT2vBMQs7E/edit?usp=sharing) 🔒, users had difficulty finding the entire showcase list of sites that use USWDS. It was easy for them to find the showcase sites on our homepage, but not the whole list.

### Describe the solution you'd like

Add a "Show all" or similar link directly under the showcase that takes users to the full list of sites.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate feature requests.

**Comment by finekatie** [comment]
Hi all. Came across this issue while searching for something else. Would like to consider the language for this link. Is "Show all" a11y-friendly language for people with visual impairments? Wondering what else we might say to take users to the Showcase page. @amycole501 / @alex-hull  would love your feedback. If "show" is appropriate, then I'm good.

**Comment by alex-hull** [comment]
Oooh, interesting question, @finekatie. I think what we have heard previously is that using language like, "see you later" is typically ok in the blind and low vision populations. However, I would be more concerned with what "show all" is referring to for someone with a cognitive disability. They may need more context as to what is being shown. Maybe something like "show/display all resources"? cc: @amycole501

**Comment by amycole501** [comment]
I agree with Alex. "Show all" isn't going to offend the visually impaired community by and large based on the conversations I've had with many low vision and blind individuals. However, I also agree that "show all" can be ambiguous (I've felt it to be that way on some sites). Being as clear and unambiguous as possible will help everyone.

**Comment by jaclinec** [comment]
I wanted to chime in. I've read in a few places and also recently attended a talk at the AccessU conference about using certain terminology with blind and low vision folks. The talk I went to, "Straight talk: What Disabled Users Want you to Know When Designing Your Product" basically took the stance that you should use common terminology - its ok and preferred to use terms like “see a person”, “watch a movie”, “read a book," because people in the blind and low vision community use those terms too. They basically said "don’t make it weird" which I thought was funny but also insightful.

Totally open to using other terms than "Show all" if it makes it clearer, though. Like "View full list of sites that use USWDS" or something similar?

**Comment by amycole501** [comment]
I've heard the same thing as Jacline for many years. Just use whatever terminology is common and clear.
